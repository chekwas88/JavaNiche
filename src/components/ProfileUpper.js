import React from 'react';
import { View, Image, StyleSheet } from 'react-native';

const ProfileUpper = ({ avatar }) => {
  return (
    <View style={styles.upperContainer}>
      <Image style={styles.profileImage} source={{ uri: avatar }} />
    </View>
  );
};

const styles = StyleSheet.create({
  upperContainer: {
    width: '100%',
    height: '40%',
    backgroundColor: '#426474'
  },
  profileImage: {
    height: 80,
    width: 80,
    borderRadius: 40,
    position: 'absolute',
    top: '80%',
    left: '40%'
  }
});

export default ProfileUpper;
