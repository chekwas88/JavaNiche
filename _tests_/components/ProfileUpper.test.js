import React from 'react';
import { shallow } from 'enzyme';
import ProfileUpper from '../../src/components/ProfileUpper';

describe('Test for profileUpper component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<ProfileUpper />);
    expect(
      wrapper
        .find('View')
        .find('Image')
        .exists()
    ).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
