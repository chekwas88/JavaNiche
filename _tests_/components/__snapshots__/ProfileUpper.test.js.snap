// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for profileUpper component should render correctly 1`] = `ShallowWrapper {}`;
